apollo_fpga/protocol/jtag_svf.py

```diff
@@ -127,7 +127,7 @@ def _lex(self):
             else:
                 line, column = self.line_column()
                 print_args = (line, column, self.buffer[self.position:self.position + 16])
-                raise SVFParsingError("unrecognized SVF data at line {}, column {} ({}...)".format(print_args))
+                raise SVFParsingError("unrecognized SVF data at line {{}}, column {{}} ({{}}...)".format(print_args))
```

Comment:
Why do you escape the braces here?

Shouldn't we expand the `print_args` for `format()`? Something like:
```python
raise SVFParsingError("unrecognized SVF data at line {}, column {} ({}...)".format(*print_args))
```

Reply:
Oh you're totally right!

Reply:
Oh you're totally right!